bank2ynab.conf

```diff
@@ -82,6 +82,31 @@ Source CSV Delimiter = ;
 Header Rows = 0
 Input Columns = skip,Date,Memo,skip,skip,skip,Inflow
 
+[AU ANZ]
+# Transactions.csv
+Source Filename Pattern = ANZ
+Header Rows = 0
+Input Columns = Date,Inflow,Memo
+Date Format = %d/%m/%y
```

Comment:
Just checking: do these formats really have 2-digit years? 
(4-digit would be uppercase %Y)